client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/project-details.component.html

```diff
@@ -24,7 +24,10 @@
         [options]="projectTypeCode"
         [optionValueField]="'projectTypeCode'"
         [optionLabelField]="'description'"
-        [tooltip]="getCodeDescription(CodeTableKeys.PROJECT_TYPE_CODE)"
+        [tooltip]="projectTypeLocked 
+          ? 'Project Type cannot be changed after any Fiscal Activity has been Endorsed and Approved'
```

Comment:
can this be moved to a communal file that holds messages rather than being hardcoded? like constants.ts

Reply:
done